modules/admin_manual/pages/configuration/files/file_sharing_configuration.adoc

```diff
@@ -119,6 +119,8 @@ e.g. read-only to avoid creating shares with too many permissions unintentionall
 ==== Restrict users to only share with users in their groups
 
 Check this option to confine sharing within group memberships.
```

Comment:
Maybe this sentence could also be revised to be similar style to the one in the next section.
```suggestion
When this option is enabled, users can only share with other users that are members of groups that the user is a member of.
```